apps/src/code-studio/components/StatusCheckmarkIcon.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import color from '@cdo/apps/util/color';
+import {makeEnum} from '@cdo/apps/utils';
+import PropTypes from 'prop-types';
+
+export const iconStatus = makeEnum('noAction', 'success', 'failure');
+
+export default function StatusCheckmarkIcon(props) {
+  let icon;
+  switch (props.displayStatus) {
+    case iconStatus.success:
+      icon = 'check';
+      break;
+    case iconStatus.failure:
+      icon = 'close';
+      break;
+    default:
+      icon = '';
+  }
+
+  const visibility = icon.length > 0;
+
+  return (
+    <i
+      style={{
+        ...styles.checkmark,
+        visibility: visibility ? 'visible' : 'hidden'
+      }}
+      className={`fa fa-${icon}`}
+      aria-hidden="true"
```

Comment:
🎉 🎉 🎉  for aria attributes! i'm trying to better get in the habit of using these consistently, so be sure to call it out if you notice i've missed them